plugins/Login/PasswordResetter.php

```diff
@@ -220,7 +220,8 @@ public function initiatePasswordResetProcess($loginOrEmail, $newPassword)
         // get the user's login
         $user = $this->getUserInformation($loginOrEmail);
         if ($user === null) {
-            throw new Exception(Piwik::translate('Login_InvalidUsernameEmail'));
+            // throw a different exception type so can be handled/suppressed
```

Comment:
It may not be clear what "a different exception type" means here based on the comment - different to what? I'd probably just say "custom exception".

```suggestion
            // throw a custom exception type so it can be handled/suppressed
```